internal/control_test_engine/gnb/gnb.go

```diff
@@ -131,31 +133,32 @@ func InitGnbForAvaibility(conf config.Config,
 		conf.GNodeB.DataIF.Port)
 
 	// start communication with AMF (server SCTP).
+	for _, amf := range conf.AMFs {
```

Comment:
We need to be careful when running multiple AMF concurrently. Some parts of GNBContext is not concurrency-safe. We may need to add Mutexes as required in GNBContext, or only uses these functions without Mutexes from a single thread (ex. in the Init function like here)

Reply:
OK, I will try to design the mutex.

Reply:
@linouxis9 Now I use `GNBContext.amfPool (sync.Map)` to maintain the multiple GNBAmf, instead of create new variable `GNBAmfList`